core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -4105,6 +4105,12 @@ object ZIO extends ZIOCompanionPlatformSpecific with ZIOCompanionVersionSpecific
   def logAnnotate(logAnnotation: => LogAnnotation, logAnnotations: LogAnnotation*): LogAnnotate =
     logAnnotate(Set(logAnnotation) ++ logAnnotations.toSet)
 
+  /**
+   * Annotates each log in this effect with the specified log annotation.
+   */
+  def logAnnotate(logAnnotations: (String, String)*): LogAnnotate =
```

Comment:
Should we perhaps use the same pattern as above in order to avoid passing an empty sequence?

```suggestion
  def logAnnotate(logAnnotation: (String, String), logAnnotations: (String, String)*): LogAnnotate =
```

Reply:
fixed